tests/integration/examples/test_aura_example.py

```diff
@@ -0,0 +1,119 @@
+
+# Copyright (c) 2002-2020 "Neo4j,"
+# Neo4j Sweden AB [http://neo4j.com]
+#
+# This file is part of Neo4j.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pytest
+
+from contextlib import redirect_stdout
+from io import StringIO
+
+# tag::aura-import[]
+from neo4j import GraphDatabase
+import logging
+from neo4j.exceptions import ServiceUnavailable
+# end::aura-import[]
+
+from neo4j._exceptions import BoltHandshakeError
+
+
+# python -m pytest tests/integration/examples/test_hello_world_example.py -s -v
+
+# tag::aura[]
+class App:
+
+    def __init__(self, uri, user, password):
+        self.driver = GraphDatabase.driver(uri, auth=(user, password))
+
+    # Don't forget to close the driver connection when you are finished with it
+    def close(self):
+        self.driver.close()
+
+    def create_friendship(self, person1_name, person2_name):
+        with self.driver.session() as session:
+            # Write transactions allow the driver to handle retries and transient errors
+            result = session.write_transaction(self._create_and_return_friendship,
+                                               person1_name, person2_name)
+            for row in result:
+                print(f"Created friendship between: {row['p1']}, {row['p2']}")
+
+    @staticmethod
+    def _create_and_return_friendship(tx, person1_name, person2_name):
+        # To learn more about the Cypher syntax, see https://neo4j.com/docs/cypher-manual/current/
+        # The Reference Card is also a good resource for keywords https://neo4j.com/docs/cypher-refcard/current/
+        query = """
+        MERGE (p1:Person { name: $person1_name })
+        MERGE (p2:Person { name: $person2_name })
+        MERGE (p1)-[:KNOWS]->(p2)
+        RETURN p1, p2
+        """
+        result = tx.run(query, person1_name=person1_name, person2_name=person2_name)
+        try:
+            return [{"p1": row["p1"]["name"], "p2": row["p2"]["name"]}
+                    for row in result]
+        # Capture any errors along with the query and data for traceability
+        except ServiceUnavailable as exception:
+            logging.error(f"{query} raised an error: \n {exception}")
+            raise
+
+    def find_person(self, person_name):
+        with self.driver.session() as session:
+            result = session.read_transaction(self._find_and_return_person, person_name)
+            for row in result:
+                print(f"Found person: {row}")
```

Comment:
Dont use `f` string function, the test suit will fail for python 3.5

use: `print("Found person: {row}".format(row=row))`